load.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * SEO Module
+ *
+ * @package hm-platform/seo
+ */
+
+namespace HM\Platform;
+
+require_once __DIR__ . '/inc/namespace.php';
+
+add_action( 'hm-platform.modules.init', function () {
+	register_module( 'seo', __DIR__, 'SEO', [
+		'enabled' => false,
+		'redirects' => false,
```

Comment:
Any problem default redirects on?

Reply:
No, it is yet another admin menu item though. Reminds me I need to make a new release that moves the menu item to the bottom or next to tools.